### PR TITLE
print html path if savePageSource is true.

### PR DESCRIPTION
**Pull request by hyunil-shin**
#210 

```
# JUnit error message

Element not found {#hello}
Expected: visible
Screenshot: file:/C:/zz.%20My%20Works/10.%20works/UI_TEST/build/reports/tests/com/nhnent/tc/console/ui/BasicCheck/test1/1503627588218.0.png
Html: file:/C:/zz.%20My%20Works/10.%20works/UI_TEST/build/reports/tests/com/nhnent/tc/console/ui/BasicCheck/test1/1503627588218.0.html
Timeout: 10 s.
Caused by: NoSuchElementException: no such element: Unable to locate element: {"method":"css selector","selector":"#hello"}
```

### CODE REVIEW

**Comment by BorisOsipov** [comment]
Hi @hyunil-shin 
Please, fix checkstyle issues:

https://travis-ci.org/codeborne/selenide/builds/268212431?utm_source=github_status&utm_medium=notification
....
[ant:checkstyle] Running Checkstyle 6.19 on 147 files
[ant:checkstyle] [ERROR] /home/travis/build/codeborne/selenide/src/main/java/com/codeborne/selenide/ex/ErrorMessages.java:42:7: 'if' is not followed by whitespace. [WhitespaceAround]
[ant:checkstyle] [ERROR] /home/travis/build/codeborne/selenide/src/main/java/com/codeborne/selenide/ex/ErrorMessages.java:43: 'if' have incorrect indentation level 7, expected level should be 6. [Indentation]
[ant:checkstyle] [ERROR] /home/travis/build/codeborne/selenide/src/main/java/com/codeborne/selenide/ex/ErrorMessages.java:43:10: 'if' is not followed by whitespace. [WhitespaceAround]
[ant:checkstyle] [ERROR] /home/travis/build/codeborne/selenide/src/main/java/com/codeborne/selenide/ex/ErrorMessages.java:44: 'if' child have incorrect indentation level 10, expected level should be 8. [Indentation]
[ant:checkstyle] [ERROR] /home/travis/build/codeborne/selenide/src/main/java/com/codeborne/selenide/ex/ErrorMessages.java:45: 'if rcurly' have incorrect indentation level 7, expected level should be 6. [Indentation]
[ant:checkstyle] [ERROR] /home/travis/build/codeborne/selenide/src/main/java/com/codeborne/selenide/ex/ErrorMessages.java:46: 'if' child have incorrect indentation level 7, expected level should be 6. [Indentation]
[ant:checkstyle] [ERROR] /home/travis/build/codeborne/selenide/src/main/java/com/codeborne/selenide/ex/ErrorMessages.java:48:6: 'else' is not preceded with whitespace. [WhitespaceAround]
[ant:checkstyle] [ERROR] /home/travis/build/codeborne/selenide/src/main/java/com/codeborne/selenide/ex/ErrorMessages.java:48:6: '}' is not followed by whitespace. [WhitespaceAround]
[ant:checkstyle] [ERROR] /home/travis/build/codeborne/selenide/src/main/java/com/codeborne/selenide/ex/ErrorMessages.java:49: 'else' child have incorrect indentation level 7, expected level should be 6. [Indentation]

**Comment by hyunil-shin** [comment]
@BorisOsipov  Thank you for your feedback.

I got many "Missing a Javadoc comment" errors in my environments (gradle check). How can I get the same result as Travis CI?
```
[ant:checkstyle] [ERROR] E:\selenide\src\main\java\com\codeborne\selenide\impl\Html.java:10:3: Missing a Javadoc comment. [JavadocMethod]
[ant:checkstyle] [ERROR] E:\selenide\src\main\java\com\codeborne\selenide\impl\Html.java:14:3: Missing a Javadoc comment. [JavadocMethod]
[ant:checkstyle] [ERROR] E:\selenide\src\main\java\com\codeborne\selenide\impl\Html.java:18:3: Missing a Javadoc comment. [JavadocMethod]
[ant:checkstyle] [ERROR] E:\selenide\src\main\java\com\codeborne\selenide\impl\Html.java:22:3: Missing a Javadoc comment. [JavadocMethod]
[ant:checkstyle] [ERROR] E:\selenide\src\main\java\com\codeborne\selenide\impl\Html.java:26:3: Missing a Javadoc comment. [JavadocMethod]
```

**Comment by BorisOsipov** [comment]
@hyunil-shin yes it's windows issue :(. On linux gradle check works well.

File: src/main/java/com/codeborne/selenide/ex/ErrorMessages.java [hunk 1]
```diff
@@ -39,7 +39,15 @@ public static String screenshot(String screenshotPath) {
     if (!Configuration.screenshots) {
       return "";
     }
-    return "\nScreenshot: " + screenshotPath;
+    if (Configuration.savePageSource) {
```
Comment (BorisOsipov) [inline src/main/java/com/codeborne/selenide/ex/ErrorMessages.java]: if Configuration.screenshots=false and Configuration.savePageSource=true
we will not have PageSource in error message. I guess it's wrong.

Comment (BorisOsipov) [reply src/main/java/com/codeborne/selenide/ex/ErrorMessages.java]: Never mind. I look through code and understand why it is so.

**Comment by BorisOsipov** [comment]
@hyunil-shin Thank you!

**Pull request merged**